maas/client/facade.py

```diff
@@ -230,6 +230,14 @@ def version(origin):
         }
 
     @facade
+    def vlans(origin):
+        return {
+            "create": origin.Vlans.create,
+            "get": origin.Vlan.read,
+            "list": origin.Vlans.read,
+        }
```

Comment:
This should not be in the Facade. VLAN's are always nested under Fabrics, so you must go to the fabric before you can get to the vlans.